### PR TITLE
[~] modification for path states in Section 4.3

### PR DESCRIPTION
**Pull request by Yanmei-Liu**
Fix issue #112 and #115

### CODE REVIEW

File: draft-ietf-quic-multipath.md [hunk 1]
```diff
@@ -484,7 +484,8 @@ guarantee that packets will actually reach the peer.
 
 The endhost can use all the paths in the "Active" state, provided
 that the congestion control and flow control currently allow sending
-of new data on a path.
+of new data on a path. Note that if a path became idle timeout,
```
Comment (mirjak) [inline draft-ietf-quic-multipath.md:487]:
```suggestion
of new data on a path. Note that if a path became idle due to a timeout, 
```

File: draft-ietf-quic-multipath.md [hunk 2]
```diff
@@ -493,8 +494,13 @@ the acknowledgment of the PATH_ABANDONED frame before moving the path
 to the "Closed" state to ensure a graceful termination of the path.
 
 When a path reaches the "Closed" state, the endhost releases all the
-path's associated resources. Consequently, the endhost is not able to
-send nor receive packets on this path anymore.
+path's associated resources, including the associated Connection IDs. 
+Endpoints SHOULD send RETIRE_CONNECTION_ID frames for releasing the
```
Comment (mirjak) [inline draft-ietf-quic-multipath.md:498]:
```suggestion
Endpoints SHOULD send RETIRE_CONNECTION_ID frames before releasing the
```

File: draft-ietf-quic-multipath.md [hunk 3]
```diff
@@ -493,8 +494,13 @@ the acknowledgment of the PATH_ABANDONED frame before moving the path
...
+associated Connection IDs following {{QUIC-TRANSPORT}}. Considering
```
Comment (mirjak) [inline draft-ietf-quic-multipath.md:499]:
```suggestion
associated Connection IDs following {{Section 19.16. of QUIC-TRANSPORT}}. Considering 
```

**Review by mirjak — APPROVED** [review]

File: draft-ietf-quic-multipath.md [hunk 4]
```diff
@@ -491,8 +492,13 @@ the acknowledgment of the PATH_ABANDONED frame before moving the path
 to the "Closed" state to ensure a graceful termination of the path.
 
 When a path reaches the "Closed" state, the endhost releases all the
-path's associated resources. Consequently, the endhost is not able to
-send nor receive packets on this path anymore.
+path's associated resources, including the associated Connection IDs.
+Endpoints SHOULD send RETIRE_CONNECTION_ID frames for releasing the
+associated Connection IDs following {{QUIC-TRANSPORT}}. Considering
+endpoints SHOULD NOT send packets on the current path in the "Closed"
```
[the remark this answers is not in the document]
Comment (mirjak) [reply draft-ietf-quic-multipath.md:498]: Actually this should not be normative here as this is just the reasoning. So maybe s/SHOULD NOT/are not expected to/

**Review by qdeconinck — APPROVED** [review]

**Pull request merged**